lib/vanagon/project/dsl.rb

```diff
@@ -157,6 +157,20 @@ def generate_source_artifacts(source_artifacts)
         @project.source_artifacts = source_artifacts
       end
 
+      # Generate os-specific packaging artifacts (rpm, deb, etc)
+      #
+      # @param pkg [Boolean] whether or not to output packages
+      def generate_packages(pkg)
+        @project.generate_packages = pkg
+      end
+
+      # Output os-specific archives containing the binary output
+      #
+      # @param archive [Boolean] whether or not to output archives
+      def generate_archives(archive)
+        @project.compiled_archive = archive
+      end
```

Comment:
Are these two used anywhere?

Reply:
These are needed to be able to set those in vanagon projects.

Reply:
ahh okay. 👍